server/node.go

```diff
@@ -244,6 +244,7 @@ func (n *Node) start(rpcServer *rpc.Server, engines []engine.Engine,
 	}
 
 	// Initialize publisher for Node Events.
+	n.status.SetNodeID(n.Descriptor.NodeID)
```

Comment:
Could you add to the comment that the NodeID has been initialized since `initStores` happens previously.